### PR TITLE
adding abbreviations

### PR DESCRIPTION
**Pull request by megbalk**
see issue #559

### CODE REVIEW

**Comment by bvarner-ebi** [comment]
> @bvarner-ebi, if I understood [your comment](https://github.com/obophenotype/uberon/issues/2363#issuecomment-1446928901) well, the additions in this PR should be RELATED instead of EXACT?

@anitacaron, @megbalk, yes, my understanding is that abbreviations should be added as RELATED synonyms with synonym type = abbreviation.

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -168146,6 +168146,7 @@ intersection_of: part_of UBERON:0001710 ! lower jaw region
 id: UBERON:0016454
 name: upper central secondary incisor tooth
 subset: pheno_slim
+synonym: "I1" EXACT ABBREVIATION []
```
Comment (anitacaron) [inline src/ontology/uberon-edit.obo:168149]: Please, change all entries to RELATED synonyms.

**Review by anitacaron — CHANGES_REQUESTED** [review]

**Comment by megbalk** [comment]
@anitacaron changed all to related. Please let me know if you see any errors.

**Comment by anitacaron** [comment]
@megbalk, thank you! I have a side question. Are you using the new Protégé?

**Comment by megbalk** [comment]
@anitacaron yep! I just confirmed with @bvarner-ebi

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -172041,6 +172043,7 @@ replaced_by: UBERON:0016929
 id: UBERON:0018281
 name: lower deciduous premolar 5
 def: "Fifth premolar (counting anteroposteriorly ) in the lower jaw." [PHENOSCAPE:alex]
+synonym: "dp5" EXACT ABBREVIATION []
```
Comment (anitacaron) [inline src/ontology/uberon-edit.obo:172046]: You still need to change this and some others.

**Review by anitacaron — CHANGES_REQUESTED** [review]

**Comment by anitacaron** [comment]
@megbalk, to confirm, did you want to remove some of the related synonyms?

**Comment by megbalk** [comment]
> 

@anitacaron yes! some of them were erroneous

**Review by anitacaron — APPROVED** [review]

**Pull request merged**